src/index.js

```diff
@@ -95,7 +95,11 @@ const getUnvisit = (entities) => {
       return unvisitEntity(input, schema, unvisit, getEntity, cache);
     }
 
-    return schema.denormalize(input, unvisit);
+    if (typeof schema.denormalize === 'function') {
```

Comment:
in what case does this function not exist? All schema types should have a `denormalize` function

Reply:
when typeof schema === 'string' | 'number' | 'symbol'

Reply:
perhaps I should remove lines 90-92 to make the primitive types be consistent? I feel the fallback is more general because there's probably typeofs we can't even anticipate - like symbol didn't used to exist.

Reply:
@paularmstrong ping